src/instructlab/configuration.py

```diff
@@ -444,6 +446,7 @@ def get_default_config() -> Config:
             model=DEFAULTS.DEFAULT_MODEL,
         ),
         generate=_generate(
+            pipeline=DEFAULTS.SDG_PIPELINE,
```

Comment:
nit: I think this is redundant, the model already has default set on the attribute. This will be used by default.

Reply:
agree w/ this. Better to just remove this line unless it's providing a different pipeline

Reply:
Removed